src/utils.ts

```diff
@@ -99,10 +99,37 @@ export function extractAccessToken(authHeader: string): string {
  * @param req
  * @returns {function(any): undefined}
  */
-export function setScopes(req: express.Request) {
+export function setTokeninfo(req: express.Request) {
   return function(data: any) {
+
+    /* tslint:disable */
+    const {
+      uid,
+      scope,
+      grant_type,
+      authorization_code,
+      cn,
+      realm,
+      token_type,
+      expires_in,
+      client_id
+    } = data;
+
+    const tokeninfo = {
```

Comment:
remove `grant_type`, `authorization_code`, `realm` and `client_id` and `token_type`